packages/frontend-test-utils/src/app/createExtensionTester.tsx

```diff
@@ -245,34 +190,50 @@ export class ExtensionTester {
     return new ExtensionQuery(node);
   }
 
+  element(): JSX.Element {
```

Comment:
Thoughts on having this be `reactElement()` instead, to match `coreExtensionData.reactElement`?